src/webapp/components/current-call/overview/Overview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { CtaButtons } from "./CtaButtons";
+import { glassColors } from "../../../pages/app/themes/dhis2.theme";
+import { CurrentStatus } from "./CurrentStatus";
+
+interface OverviewProps {
+    moduleName: string;
+}
+
+export const Overview: React.FC<OverviewProps> = ({ moduleName }) => {
+    const [screen, setScreen] = useState<string>("status");
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We typically remove unused code, if it needs to be recover, it's in the history.

Reply:
Noted Arnau: I added an update here: https://github.com/EyeSeeTea/glass-dev/commit/e61d7e279992c7e10eb627ce98de9548ab0c2ad0. 
I just added a TODO: comment so I'll get back to it. Is this okay or should I just stash the state and renderScreen function for now and only add when there's actual usage implemented/

Reply:
> We typically remove unused code, if it needs to be recover, it's in the VCS
- noted on this one @tokland